x-pack/legacy/plugins/siem/public/store/network/selectors.ts

```diff
@@ -21,70 +33,79 @@ export const dnsSelector = () =>
     selectNetworkPage,
     network => network.queries.dns
   );
-export enum NetworkTableType {
-  tls = 'tls',
-  dns = 'dns',
-  topCountriesDestination = 'topCountriesDestination',
-  topCountriesSource = 'topCountriesSource',
-  topNFlowDestination = 'topNFlowDestination',
-  topNFlowSource = 'topNFlowSource',
-}
 
-export const topNFlowSelector = (flowTarget: FlowTargetSourceDest, networkType: NetworkType) => {
-  if (networkType === NetworkType.page) {
-    return createSelector(
-      selectNetworkPage,
-      network =>
-        flowTarget === FlowTargetSourceDest.source
-          ? network.queries[NetworkTableType.topNFlowSource]
-          : network.queries[NetworkTableType.topNFlowDestination]
-    );
-  }
-  return createSelector(
-    selectNetworkDetails,
-    network =>
-      flowTarget === FlowTargetSourceDest.source
-        ? network.queries[IpDetailsTableType.topNFlowSource]
-        : network.queries[IpDetailsTableType.topNFlowDestination]
+const selectTopNFlowByType = (
+  state: State,
+  networkType: NetworkType,
+  flowTarget: FlowTargetSourceDest
+) => {
+  const ft = flowTarget === FlowTargetSourceDest.source ? 'topNFlowSource' : 'topNFlowDestination';
+  const nFlowType =
+    networkType === NetworkType.page ? NetworkTableType[ft] : IpDetailsTableType[ft];
+  return (
+    get([networkType, 'queries', nFlowType], state.network) || {
+      activePage: DEFAULT_TABLE_ACTIVE_PAGE,
+      limit: DEFAULT_TABLE_LIMIT,
+      sort: {
+        field: NetworkTopTablesFields.bytes_out,
+        direction: Direction.desc,
+      },
```

Comment:
I wonder if for the fallback default values here we should be referencing the initial state values from the reducer so there's no possibility of one changing without the other?

e.g.

``` ts
get([networkType, 'queries', nFlowType], state.network) || get([networkType, 'queries', nFlowType], initialNetworkState) 
```